unit-tests/src/test/java/org/eclipse/collections/impl/AbstractRichIterableTestCase.java

```diff
@@ -1841,6 +1841,13 @@ public void makeStringWithSeparatorAndStartAndEnd()
         Assert.assertEquals(collection.toString(), collection.makeString("[", ", ", "]"));
     }
 
+    @Test
+    public void fusedCollectMakeString()
+    {
+        RichIterable<Integer> collection = this.newWith(1, 2, 3);
+        Assert.assertEquals(collection.toString(), collection.makeString(Integer::toUnsignedString, "[", ", ", "]"));
```

Comment:
Please make this consistent with the `RichIterableTestCase` test above using `asLazy().collect().makeString()`.

Reply:
Ops - my bad, fixing